src/main/java/core/basesyntax/service/ReportServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import java.util.stream.Collectors;
+
+public class ReportServiceImpl implements ReportService {
```

Comment:
```suggestion
public class CsvReportServiceImpl implements ReportService {
```
putting csv to the name of service makes sense because this service makes report in this format